.github/workflows/user_yaml_ci.yaml

```diff
@@ -58,7 +59,11 @@ jobs:
           git checkout localbranch
         fi
 
-        # lint the files
+        # lint the files using yamllint
+        yamllint ./users/anvilstaging/*
+        yamllint ./users/datastageinternalstaging/*
+        yamllint ./users/stagingdatastage/*
```

Comment:
can we run it on all of them instead of those 3?

Reply:
I guess we should also fix those `too many spaces` issue? 😅 https://github.com/uc-cdis/commons-users/actions/runs/10814267330/job/30000314892

Reply:
yes 🤔 let's fix the errors and ignore the warnings